src/approaches/bilevel_planning_approach.py

```diff
@@ -105,6 +93,22 @@ def reset_metrics(self) -> None:
         # Initialize min to inf (max gets initialized to 0 by default).
         self._metrics["min_num_skeletons_optimized"] = float("inf")
 
+    def _save_metrics(self, metrics: Metrics, nsrts: Set[NSRT],
+                      predicates: Set[Predicate]) -> None:
+        for metric in [
+                "num_skeletons_optimized", "num_failures_discovered",
+                "num_nodes_expanded", "num_nodes_created", "plan_length"
+        ]:
```

Comment:
oh there's an indent problem here .... @tomsilver this was in #1097 too. is it a bug there?

Reply:
oops yeah good catch